### DISCUSSION TITLE
Creating IPF color triangle using orientation input

### DISCUSSION THREAD

**Discussion by LiZa111111111** [neperfepx/neper#570] [Q&A]
Hello.
Inverse pole figures should look like something below, see the figure. 
Is it possible to generate them considering 3 planes normal to X, Y and Z directions? I this case it would give a better understanding of the texture along X or Y or Z direction. Thanks.
![image](https://user-images.githubusercontent.com/113169470/192718632-e210d16d-0754-401f-b5e9-339211592c70.png)

**Comment by rquey** [comment]
It will be possible in the next release.
